.coderabbit.yaml

```
@@ -0,0 +1,15 @@
+# .coderabbit.yaml
+ language: "ko-KR" # 한국말로 설정
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 1

(indentation)